loadtest/app.go

```diff
@@ -0,0 +1,175 @@
+package loadtest
+
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+	"time"
+
+	"github.com/spf13/viper"
+	"github.com/topfreegames/khan/lib"
+	"github.com/topfreegames/khan/log"
+	"github.com/uber-go/zap"
+)
+
```

Comment:
I think a somewhat detailed description of how loadtest.App works is very welcome. For example:
- What's the test flow?
- Everything that's configurable and what it's names mean.

Reply:
Sure! I'll write a README.md

Reply:
Done!